crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -120,6 +121,71 @@ pub struct ExecutionIndicesWithHash {
     pub hash: u64,
 }
 
+/// ConsensusStats is versioned because we may iterate on the struct, and it is
+/// stored on disk.
+#[enum_dispatch]
+pub trait ConsensusStatsAPI {
```

Comment:
This type may grow in code, may be good to put in a separate file

Reply:
Do we want to create a file to hold basic types from sui core authority, or just put this type in its own file? I think it will at most add a few more entries, e.g. total gas submitted by a validator.

Reply:
We can refactor later when the struct grows.